app/jobs/pusher_job.rb

```diff
@@ -1,7 +1,8 @@
 class PusherJob  < ActiveJob::Base
   queue_as :default
 
-  def perform(subarray_of_channels, event, data)
-    Pusher.trigger(subarray_of_channels, event, data)
+  def perform(channels, event, data)
+    url = Rails.application.secrets.socketio_service["url"]
+    Nestful.post url, {rooms:channels, event:event, args:JSON.parse(data)}, :format => :json
```

Comment:
Line is too long. [92/80]<br>Use the new Ruby 1.9 hash syntax.<br>Space missing after colon.<br>Space inside { missing.<br>Space inside } missing.